pvm.py

```diff
@@ -38,74 +39,90 @@ def _init_logger():
 IS_FILE_SET = False
 
 def parse_commands(*args):
-	global media
-	global VIDEO_PATH
-	global IS_FILE_SET
-	command = args[1]
-	_logger.info("Received command: %s", command)
-	if len(args)>2:
-		value = args[2]
-		_logger.info("Received command: %s", str(value))
-		pass
-	# TODO: Create another python file to control two display
-	try:
-		if command=="file":
-			_logger.info("File set: %s", PEFIX_PATH + value)
-			IS_FILE_SET = True
-			media = OMXPlayer(PEFIX_PATH + value, dbus_name='org.mpris.MediaPlayer2.omxplayer', args=['--loop'])
-			media.pause()
-			VIDEO_PATH = value
-			return
+    global media
+    global VIDEO_PATH
+    global IS_FILE_SET
+    command = args[1]
+    _logger.info("Received command: %s", command)
+    if len(args)>2:
+        value = args[2]
+        _logger.info("Received command: %s", str(value))
+        pass
+    # TODO: Create another python file to control two display
+    try:
+        if command=="file":
+            _logger.info("File set: %s", PEFIX_PATH + value)
+            IS_FILE_SET = True
+            media = OMXPlayer(PEFIX_PATH + value, dbus_name='org.mpris.MediaPlayer2.omxplayer', args=['--loop'])
+            media.pause()
+            VIDEO_PATH = value
+            return
 
-		if not IS_FILE_SET:
-			_logger.info("Command %s failed because of the file is unset.", command)
-			return
+        if not IS_FILE_SET:
+            _logger.info("Command %s failed because of the file is unset.", command)
+            return
```

Comment:
I would suggest reverting this file and then making your changes so that the indentation will not be a problem.
Correct me if I am wrong.

Reply:
yeah, Imma bit lost ngl. 
I like a good indentation but it needs to be its own PR.
Right now it is difficult to see What has changed here.
I'll leave you 2 to figure this one out 😅